tests/cucumber/steps/steps.js

```diff
@@ -608,40 +608,6 @@ module.exports = function getSteps(options) {
     return this.acl.ledgerSupply();
   });
 
-  Then('I get transactions by address and round', async function () {
```

Comment:
I'm surprised this is passing because I still see the step [here](https://github.com/algorand/algorand-sdk-testing/blob/3d2db01121025d8b531d30b83ad1cfa23e5e65c4/features/integration/algod.feature#L22).

Reply:
Good point - I thought we would remove it because it uses the `transactionByAddress` v1 endpoint, but I guess it doesn't invoke the v1 indexer so it should be kept

I'll also try to dig into what is wrong

Reply:
The feature files were being copied to the wrong location - this should be fixed in the [latest commit](https://github.com/algorand/js-algorand-sdk/pull/623/commits/cc0411a834e444f59d1796147d7e1d2c6bed954a)

Reply:
Build: https://app.circleci.com/pipelines/github/algorand/js-algorand-sdk/449/workflows/25e2996f-d027-44af-a2c5-a126832539af/jobs/2547